### DISCUSSION TITLE
Focus issues

### DISCUSSION THREAD

**Discussion by KaKi87** [coder/code-server#6768] [Q&A]
Hi,

I'm having focus issues :

- when searching text in file, file editor takes focus after each key press in search input ;
- when terminal is active, file editor only focuses after clicking it twice.

Also, sometimes, a second cursor appears in file editor, but in a dotted form, at some distance in the top-left direction of the actual cursor, triggering scroll when reaching the edges of the editor.

I'm running the last version (4.23.1).

Thanks

**Comment by code-asher** [comment]
Are you able to reproduce in vscode.dev or Codespaces?  It sounds likely to be an issue we can file upstream: https://github.com/microsoft/vscode

**Reply by KaKi87** [reply]
Nope, and I didn't have these issues a few code-server versions ago :/

**Reply by code-asher** [reply]
Ah it does not happen in either of those?  Very interesting...we are not making any UX changes, for the most part we just wrap VS Code as it is, so I am not sure what we could have done to break it.  :thinking: 

I tried reproducing in 4.23.1 (using Chromium) but neither of those focus issues are reproducing for me.
